src/libs/logos.js

```diff
@@ -12,10 +12,8 @@ import fcLogoWhite from 'src/images/brands/firecloud/FireCloud-Logo-White.svg'
 import terraLogo from 'src/images/brands/terra/logo.svg'
 import terraLogoWhite from 'src/images/brands/terra/logo-grey.svg'
 import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
-import ukbLogo from 'src/images/brands/ukbiobank/uk-biobank-logo-color.svg'
-import ukbLogoWhite from 'src/images/brands/ukbiobank/uk-biobank-logo-white.svg'
```

Comment:
nit: also delete the `src/images/brands/ukbiobank/` directory

Reply:
good catch!